### DISCUSSION TITLE
Is there an ETA on Appium 2.0 leaving beta?

### DISCUSSION THREAD

**Discussion by kevin-roe** [appium/appium#17893] [General]
My company is very waring of using beta software, for obvious reasons.  However, with the upgrade to XCode 14, I see now that 1.22 is no longer supported, so now we need to upgrade to appium 2.0 beta on all new machines that come with Xcode 14.  Do you have a timeline on when 2.0.0 will leave beta?  is there a substantial reason it's still in beta?  I feel it's quite stable, so I'm not hugely worried about it, but others I work with are.  Thank you!

**Comment by jlipps** [comment]
The only reason it's still in beta is because the documentation is not yet complete. We don't want to say it's launched without the proper documentation in place. We're getting very close. But in terms of actual behaviour and stability, yeah it's been a very long time since we've recommended people switch over to Appium 2. There's a very robust [migration guide](https://appium.github.io/appium/docs/en/2.0/guides/migrating-1-to-2/) in the new docs. That should be everything you need to be aware of.

**Comment by cervebar** [comment]
I'm inquiring if there have been any recent developments regarding the exit of Appium 2 from its beta stage. Do you have an estimated timeline for when this transition will occur? Our situation involves developing software that must adhere to strict Regulatory Compliance standards, which prohibits the use of beta-stage software. Although Appium 2 appears to be stable and suitable for our requirements, the fact that it is still in beta is a significant obstacle for us. We genuinely desire to utilize Appium 2, but this constraint is currently preventing us from doing so. Your assistance would be greatly appreciated. 🙏🏻

**Reply by jlipps** [reply]
within the next few weeks
